addons/reinforceToHC/functions/fnc_getSpawnMachineId.sqf

```diff
@@ -0,0 +1,33 @@
+#include "script_component.hpp"
+#define SERVER_CLIENT_ID 2
+
+private _returnClientId = SERVER_CLIENT_ID;
+private _allHCs = entities "HeadlessClient_F";
+TRACE_1("",_allHCs);
+
+if (_allHCs isEqualTo []) exitWIth { _returnClientId };
+
+//Find the HC with least units on it
+private _hcIDs = [];
+private _hcCounts = [];
+
+{
+    _hcIDs pushBack (owner _x);
+    _hcCounts pushBack 0;
+    nil
+} count _allHCs;
+
+{
+    private _index = _hcIDs find (groupOwner _x);
+    if (_index > 0) then {
```

Comment:
This compare needs to be _index > -1. It's skipping index 0 (first HC) so it's not adding any of the units.